src/netspresso_trainer/schedulers/builder.py

```diff
@@ -5,18 +5,14 @@
 
 
 def build_scheduler(optimizer, conf_training):
-    scheduler_name = conf_training.sched
+    conf_scheduler = conf_training.scheduler
+    scheduler_name = conf_scheduler.name
```

Comment:
I suggest changing `conf_scheduler` to `scheduler_conf` in my opinion.
```suggestion
    scheduler_conf = conf_training.scheduler
    scheduler_name = scheduler_conf.name
```